core/node/eth_sender/README.md

```diff
@@ -0,0 +1,34 @@
+# Eth-sender `zksync_eth_sender`
+
+Eth-sender is the component that executes scheduled transactions on Ethereum. As of 06.2024 it is only used by core to
+execute Commit, Prove and Execute L1 transactions.
+
+It was designed with fire-and-forget ideology in mind, as soon as you send transaction to be executed, eth-sender will
+make sure it is mined, handling stuff as bumping gas prices and resending transaction if needed.
+
+## API
+
+eth-sender API exposes only two methods:
```

Comment:
There will be also a second, Admin API usable via CLI to make common operational tasks less error-prone, but it will be designed separately